### PR TITLE
ENH: DataFrame __divmod__, __rdivmod__

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -5960,6 +5960,18 @@ def _construct_result(self, result) -> DataFrame:
         out.index = self.index
         return out
 
+    def __divmod__(self, other):
```
Comment (jreback) [inline pandas/core/frame.py:5963]: shouldn't these be with other arith operators?

Comment (jreback) [inline pandas/core/frame.py:5963]: can you type

**Review by jreback — CHANGES_REQUESTED** [review]
is this new? or just a better definition for the op? (e.g. do you need a whatsnew)

File: pandas/core/frame.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/core/frame.py:5963]: this is the arithmetic section

Comment (jbrockmendel) [reply pandas/core/frame.py:5963]: sure

**Comment by jbrockmendel** [comment]
> is this new? or just a better definition for the op? (e.g. do you need a whatsnew)

new, will add whatsnew

**Review by jreback — APPROVED** [review]

**Pull request merged**